### PR TITLE
Use httpexpect on integration tests for web API

### PR DESCRIPTION
**Pull request by marcosanchotene**

### CODE REVIEW

File: test/objectstorage_tests/go.mod [hunk 1]
```diff
@@ -9,13 +9,41 @@ require (
 	golang.org/x/oauth2 v0.5.0
 )
 
+require (
```
Comment (jeroiraz) [inline test/objectstorage_tests/go.mod:12]: better to use `document` naming everywhere
e.g. `module github.com/codenotary/immudb/test/document_storage_tests`

even in the folders e.g. `tests/document_storage_tests` and `document_storage_tests`

File: test/objectstorage_tests/objects_tests/create_collections_test.go [hunk 2]
```diff
@@ -0,0 +1,319 @@
+package main
```
Comment (jeroiraz) [inline test/objectstorage_tests/objects_tests/create_collections_test.go:1]: copyright notice

File: test/objectstorage_tests/objects_tests/create_collections_test.go [hunk 3]
```diff
@@ -0,0 +1,319 @@
...
+)
+
+type CreateCollectionsTestSuite struct {
+	suite.Suite
+	expect *httpexpect.Expect
+	token  string
+}
+
+func (s *CreateCollectionsTestSuite) SetupTest() {
+	baseURL, exists := os.LookupEnv("OBJECTS_TEST_BASEURL")
```
Comment (jeroiraz) [inline test/objectstorage_tests/objects_tests/create_collections_test.go:22]: `DOCUMENTS_TEST_BASEURL`

File: test/objectstorage_tests/objects_tests/create_collections_test.go [hunk 4]
```diff
@@ -0,0 +1,319 @@
...
+	}
+
+	user := map[string]interface{}{
+		"username": "immudb",
+		"password": "immudb",
+		"database": "defaultdb",
+	}
+
+	s.expect = httpexpect.Default(s.T(), baseURL)
+	obj := s.expect.POST("/authorization/session/open").
```
Comment (jeroiraz) [inline test/objectstorage_tests/objects_tests/create_collections_test.go:35]: `doc := ...`

Comment (jeroiraz) [reply test/objectstorage_tests/objects_tests/create_collections_test.go:35]: however in this case expected response is not a document...

**Review by jeroiraz — APPROVED** [review]
LGTM, made comments related to `document` naming

File: test/objectstorage_tests/go.mod (hunk 1 above)
Comment (marcosanchotene) [reply test/objectstorage_tests/go.mod:12]: This was already like that, but I'm changing it.

**Pull request merged**